### PR TITLE
Adding an example of setting up local embedding model

### PR DESCRIPTION
**Pull request by ZiTao-Li**
---
name: Adding an example of setting up local embedding model
about: as title
---
## Description

As title says:
* provide an example in the tutorial for setting up a local embedding model based on `sentence_transformers`, which supports both HuggingFace and ModelScope models.
* provide an example of how to use PostAPIEmbeddingWrapper

Related to https://github.com/modelscope/agentscope/issues/315

## Checklist

Please check the following items before code is ready to be reviewed.

- [ ]  Code has passed all tests
- [ ]  Docstrings have been added/updated in Google Style
- [ ]  Documentation has been updated
- [ ]  Code is ready for review

### CODE REVIEW

File: docs/sphinx_doc/en/source/tutorial/210-rag.md [hunk 1]
```diff
@@ -190,6 +190,109 @@ RAG agent is an agent that can generate answers based on the retrieved knowledge
 Your agent will be equipped with a list of knowledge according to the `knowledge_id_list`.
 You can decide how to use the retrieved content and even update and refresh the index in your agent's `reply` function.
 
+## (Optional) Setting up a local embedding model service
+
+For those who are interested in setting up a local embedding service, we provide the following example based on the
+`sentence_transformers` package, which is a popular specialized package for embedding models (based on the `transformer` package and compatible with both HuggingFace and ModelScope models).
+In this example, we will use one of the SOTA embedding models, `gte-Qwen2-7B-instruct`.
+
+* Step 1: follow the instruction on [HuggingFace](https://huggingface.co/Alibaba-NLP/gte-Qwen2-7B-instruct) or [ModelScope](https://www.modelscope.cn/models/iic/gte_Qwen2-7B-instruct ) to download the embedding model.
```
Comment (DavdGao) [inline docs/sphinx_doc/en/source/tutorial/210-rag.md:199]: ==> Follow (upper case)

Comment (DavdGao) [reply docs/sphinx_doc/en/source/tutorial/210-rag.md:199]: Maybe we can add guidance for huggingface mirror by 
```bash
export HF_ENDPOINT=https://hf-mirror.com
```
or 
```python
os.environ["HF_ENDPOINT"] = "https://hf-mirror.com"
```

**Review by DavdGao — COMMENTED** [review]
Please see inline comments, others lgtm.

File: docs/sphinx_doc/en/source/tutorial/210-rag.md (hunk 1 above)
Comment (ZiTao-Li) [reply docs/sphinx_doc/en/source/tutorial/210-rag.md]: updated

**Review by DavdGao — APPROVED** [review]
lgtm

**Pull request merged**